pkgs/development/python-modules/xdot/default.nix

```
@@ -13,7 +13,18 @@ buildPythonPackage rec {
   disabled = !isPy3k;
 
   nativeBuildInputs = [ wrapGAppsHook ];
-  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 ];
+  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 numpy ];
+  checkInputs = [ xvfb_run ];
+
+  checkPhase = ''
+    cat >test-runner <<EOF
+    #!/bin/sh
+
+    ${python.pythonForBuild.interpreter} nix_run_setup test
+    EOF
+    chmod +x test-runner
+    xvfb-run -s '-screen 0 800x600x24' ./test-runner
```

```suggestion
    xvfb-run -s '-screen 0 800x600x24' ${python.pythonForBuild.interpreter} nix_run_setup test
```

Can we just run it like this? Can we use python3?

---

ah neat, [that](https://github.com/NixOS/nixpkgs/compare/091fabd2e07e6f2efbfc9b389fa31f0839e51243..9a81e382c4a92ff517f390273665d3427bfc00b6) is better!